contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -1519,7 +1519,7 @@ UNION
 SELECT
 CAST(t2.dbname AS sys.sysname) AS TABLE_QUALIFIER,
 CAST(s1.name AS sys.sysname) AS TABLE_OWNER,
-CAST(t1.relname AS sys.sysname) AS TABLE_NAME,
+CAST(t1.relname AS sys.sysname) COLLATE sys.database_default AS TABLE_NAME,
```

Comment:
Curious, why does this one have COLLATE sys.database_default but not the other fields? I would assume since it's casted to sys.sysname the collation would also apply since it's a sys datatype?

Reply:
Yes it was a redundant collate clause fixed in the latest revision